lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
 
     // GET or HEAD for weak freshness validation only
-    if ('GET' != method && 'HEAD' != method) return false;
+    if (!['GET', 'HEAD'].includes(method)) return false;
```

Comment:
And? It's will be whole native code, easily extensible in case of... Way more readable. Any benchmarks?